data-otservbr-global/npc/gnombold.lua

```diff
@@ -146,15 +146,15 @@ local function creatureSayCallback(npc, creature, type, message)
 	////GEOMANTIC CHARGES////
```

Comment:
These types of comments can also be removed, they don't help at all and make the code ugly.